<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CachePolicyPage</class>
 <widget class="QWidget" name="CachePolicyPage">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>461</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="verticalLayout_3">
   <item row="0" column="1">
    <widget class="QCheckBox" name="inherit">
     <property name="text">
      <string>Use options from parent folder or account</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="syncOnDemand">
     <property name="text">
      <string>Synchronize when selecting this folder</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Synchronize after:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="checkInterval">
     <property name="specialValueText">
      <string comment="never check the cache">Never</string>
     </property>
     <property name="suffix">
      <string>minutes</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="localPartsLabel">
     <property name="text">
      <string>Locally cached parts:</string>
     </property>
     <property name="buddy">
       <cstring>localParts</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KEditListWidget" name="localParts">
     <property name="minimumSize">
      <size>
        <width>150</width>
      </size>
     </property>
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="buttons">
      <set>KEditListWidget::Add|KEditListWidget::Remove</set>
     </property>
     <property name="title" stdset="0">
      <string>Locally Cached Parts</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="retrievalOptionsLabel">
     <property name="text">
      <string>Retrieval Options:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QRadioButton" name="retrieveFullMessages">
     <property name="text">
      <string>Always retrieve full &amp;messages</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QRadioButton" name="retrieveOnlyHeaders">
     <property name="text">
      <string>&amp;Retrieve message bodies on demand</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Keep message bodies locally for:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="localCacheTimeout">
     <property name="minimumSize">
      <size>
        <width>150</width>
      </size>
     </property>
     <property name="specialValueText">
      <string comment="no cache timeout">Forever</string>
     </property>
     <property name="suffix">
      <string>minutes</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="7">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QWidget</extends>
   <header>keditlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>inherit</sender>
   <signal>toggled(bool)</signal>
   <receiver>localParts</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>retrieveOnlyHeaders</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>retrieveOnlyHeaders</sender>
   <signal>toggled(bool)</signal>
   <receiver>localCacheTimeout</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inherit</sender>
   <signal>toggled(bool)</signal>
   <receiver>syncOnDemand</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inherit</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inherit</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkInterval</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
